package.json

```
@@ -28,7 +28,8 @@
     "build": "tsup",
     "test": "jest",
     "lint": "eslint .",
-    "prepublishOnly": "pnpm run lint && pnpm run test && pnpm run build"
+    "prepublishOnly": "pnpm run lint && pnpm run test && pnpm run build",
+    "update-manufacturers": "npx tsx scripts/generate-manufacturers-file.ts"
```

```suggestion
    "updateManufacturers": "npx tsx scripts/generate-manufacturers-file.ts"
```

Let's stick with the repository's convention, @rikkertpm 😇